### DISCUSSION TITLE
KIE  key-value 標注問題

### DISCUSSION THREAD

**Discussion by bhhsieh** [PaddlePaddle/PaddleOCR#12285] [Q&A]
KIE 任務的標注中提到
<img width="792" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/138775305/5b1b92c6-12d7-4ed1-b204-5ed75607f639">
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/ppstructure/kie/how_to_do_kie.md
支持一個 key 對應多個 value

請問支持多個 key 對應到同一個 value 嗎？

**Comment by UserWangZz** [comment]
应该是支持的，它的kv对不会在同一个linking中出现

**Comment by bhhsieh** [comment]
我將 SER 和 RE 模型 finetune 之後還是沒辦法獲得多個 key 對應到同一個 value 的狀況
想問是不是有什麼範例確定這是可行的

**Comment by UserWangZz** [comment]
目前暂时还没有相关的范例，这个场景在数据集中出现频繁吗？

**Comment by bhhsieh** [comment]
在我們的數據集蠻常出現的，但 inference 的結果發現都無法獲得多個 key 對應到同一個 value 的狀況

**Comment by UserWangZz** [comment]
这种情况建议增加这种类型的数据，或者在训练的时候调大学习率尝试会不会有所改善
